draft/webxdc-user-guide.md

```diff
@@ -0,0 +1,156 @@
+# Webxdc User Guide
+
+## Webxdc File Format
+
+- a **Webxdc app** is a **ZIP-file** with the extension `.xdc`
+- the ZIP-file must contain at least the file `index.html`
+- if the Webxdc app is started, `index.html` is opened in a restricted webview
+  that allow accessing resources only from the ZIP-file
+
+
+## Webxdc API
```

Comment:
should we later move this into it's own documentation repo?
we can't really provide ts types or jsdoc types in a markdown file. (they provide documentation and auto completion in IDE's)

Reply:
that will improve and move at some point, sure.